includes/html/table/alertlog-stats.inc.php

```diff
@@ -13,7 +13,18 @@
  * @author     LibreNMS Contributors
 */
 
+$alert_severities = array(
+    // alert_rules.status is enum('ok','warning','critical')
+    'ok' => 1,
```

Comment:
moving this Array to an Enum would be appreciated
take a look on LibreNMS\Enum directory for examples